spec.html

```diff
@@ -36504,6 +37172,16 @@ <h1>get RegExp.prototype.unicode</h1>
           1. Return ? RegExpHasFlag(_R_, _cu_).
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-get-regexp.prototype.unicodesets">
+        <h1>get RegExp.prototype.unicodeSets</h1>
+        <p>`RegExp.prototype.unicodeSets` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:</p>
+        <emu-alg>
+          1. Let _R_ be the *this* value.
+          1. Let _cu_ be the code unit 0x0076 (LATIN SMALL LETTER V).
+          1. Return ? RegExpHasFlag(_R_, _cu_).
```

Comment:
i see this is the pattern used in `.unicode` but i'm not sure why both couldn't be:
```suggestion
          1. Return ? RegExpHasFlag(_R_, *"v"*).
```